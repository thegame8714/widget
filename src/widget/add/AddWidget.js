import React from 'react'
import { Title } from '../styles/'

const AddWidget = () => <Title>Add a new Widget</Title>

export default AddWidget
