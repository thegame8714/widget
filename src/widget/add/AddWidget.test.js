import renderer from 'react-test-renderer'
import React from 'react'
import AddWidget from './AddWidget'

describe('Add Widget page', () => {
  it('should render the page', () => {
    const addWidgetComponent = renderer.create(<AddWidget />).toJSON()

    expect(addWidgetComponent).toMatchSnapshot()
  })
})
