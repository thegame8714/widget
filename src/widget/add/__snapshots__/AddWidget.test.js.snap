// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add Widget page should render the page 1`] = `
<h1
  className="sc-bdVaJa ghfIaL"
>
  Add a new Widget
</h1>
`;
